docker-compose.yml

```diff
@@ -23,4 +23,4 @@ services:
    command: /usr/local/bin/gunicorn -w 4 -b 0.0.0.0:3000 app:app
    ports:
      - "3000:3000"
-     - "5432:5432"
+     - "5432:5433"
```

Comment:
@redshiftzero I think this should be "5433:5432" instead, so it's 5433 on the host.

Reply:
doh thanks